lib/kitchen/driver/openstack.rb

```diff
@@ -82,23 +81,22 @@ def create(state)
             config[:server_name] = default_name
           end
         end
-        config[:disable_ssl_validation] && disable_ssl_validation
+        disable_ssl_validation if config[:disable_ssl_validation]
```

Comment:
So it's mentioned in the README, about being turned on ONLY if you know what you're doing. I don't think it's supposed to be used or acknowledged without specific reasons.